packages/connectivity/connectivity_macos/README.md

```diff
@@ -6,25 +6,27 @@ The macos implementation of [`connectivity`].
 
 ### Import the package
 
-To use this plugin in your Flutter Web app, simply add it as a dependency in
-your `pubspec.yaml` alongside the base `connectivity` plugin.
 
-_(This is only temporary: in the future we hope to make this package an
-"endorsed" implementation of `connectivity`, so that it is automatically
-included in your Flutter macos app when you depend on `package:connectivity_macos`.)_
+This package has been endorsed, meaning that you only need to add `connectivity`
+as a dependency in your `pubspec.yaml`. It will be automatically included in your app
+when you depend on `package:connectivity`.
 
 This is what the above means to your `pubspec.yaml`:
 
 ```yaml
 ...
 dependencies:
   ...
-  connectivity: ^0.4.6
-  connectivity_macos: ^0.0.1
+  connectivity: ^0.4.8
   ...
 ```
 
-### Use the plugin
+If you wish to use the macos package only, you can add  `connectivity_macos` as a
```

Comment:
I don't think people should ever use the macos package only; this package should *only* be used through the core `connectivity` plugin.

Reply:
People might still need to import platform plugins if there are platform-unique functionalities. This would avoid having a huge front facing api with lots of `if (platform)` all over the place. We might not see that as a problem now, but once we add more platforms it will be more evident. This is part of the plan for federated plugins.